added uniforms, helmets, optics, bipods and attachments

rdbeerman opened the pull request:
fixed formatting
removed ReaperCoPilot
added binoculars to BasicItems
added BasicUniforms
added ReconUniforms
added ReaperUniforms
added HavocUniforms
removed AssaultRiflesC from HavocPilot
added BasicHelmets
added ReconHelmets
added ReaperHelmets
added HavocHelmets
added SmallOptics
added MediumOptics
added LargeOptics
added Bipods
added Suppressors
added Attachments

rdbeerman committed:
added uniforms, helmets, optics, bipods and attachments

legman commented on SpecialOperations2.Altis/sos/arsenal/VirtualArsenal.hpp:
```diff
@@ -16,9 +17,13 @@ class SOS_WestVirtualArsenal {
             "Berets",
-            "Goggles"
+            "Goggles",
+            "SmallOptics",
```
SmallOptics -> **ShortRangeOptics**

legman commented on SpecialOperations2.Altis/sos/arsenal/VirtualArsenal.hpp:
```diff
@@ -16,9 +17,13 @@ class SOS_WestVirtualArsenal {
-            "Goggles"
+            "Goggles",
+            "SmallOptics",
+            "MediumOptics",
```
MediumOptics -> **MidRangeOptics**

legman commented on SpecialOperations2.Altis/sos/arsenal/VirtualArsenal.hpp:
```diff
@@ -56,13 +67,21 @@ class SOS_WestVirtualArsenal {
+            "Goggles",
+            "SmallOptics",
+            "MediumOptics",
+            "LargeOptics",
```
LargeOptics -> **LongRangeOptics**

legman commented:
Add an empty class **MedicHelmets** and assign to all medics for when we include the S.O.S Vest mod.

``` javascript
class MedicHelmets : BasicHelmets;
```

should suffice for now and will extend the existing basic helmet definition!

legman commented on SpecialOperations2.Altis/sos/arsenal/VirtualItemCargo.hpp:
```diff
@@ -123,5 +188,52 @@ class SOS_WestVirtualItemCargo {
+        };
+    };
+    
+    class LargeOptics {
```
Move thermal optics to own class **ThermalOptics**

legman commented on SpecialOperations2.Altis/sos/arsenal/VirtualItemCargo.hpp:
```diff
@@ -1,4 +1,68 @@
+        };
+    };
+    
+    class ReconUniforms {
```
move Ghillie uniforms into own class **GhillieUniforms** and assign to recon in addition to recon uniforms!

rdbeerman committed:
changed SmallOptics to ShortRangeOptics
changed MediumOptics to MidRangeOptics
changed LargeOptics to LongRangeOptics
added placeholder MedicHelmets which follows BasicHelmets
added ThermalOptics
added GhillieUniforms

rdbeerman committed:
removed TWS from LongRangeScopes
